docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -5,70 +5,98 @@ In select
 and
 [{{brand_compliant}}](../../../reference/features/compliant.md#virtualization)
 regions, you can instantiate virtual machines --- or simply _servers_
---- with virtual GPU (vGPU) support. For that, you use the `openstack`
-CLI utility.
+--- with virtual GPU (vGPU) support. For that, you may either use the
+{{brand}} Dashboard or the `openstack` CLI utility.
 
 ## Prerequisites
 
 Although not strictly necessary, it is certainly helpful to have some
-familiarity with [creating servers in {{brand}}](new-server.md),
-especially with the OpenStack CLI.
+familiarity with [creating servers in {{brand}}](new-server.md).
 
 ## Instantiating a vGPU server
 
-Before you begin, please make sure to source [your `openstack` RC file
-first](../../getting-started/enable-openstack-cli.md).
-
-As we discuss in the [guide on creating servers](new-server.md), an
-`openstack` command for instantiating a new server looks like this:
-
-```console
-openstack server create \
-    --flavor $FLAVOR_NAME \
-    --image $IMAGE_NAME \
-    --boot-from-volume $VOL_SIZE \
-    --network $NETWORK_NAME \
-    --security-group $SEC_GROUP_NAME \
-    --key-name $KEY_NAME \
-    --wait \
-    $SERVER_NAME
-```
-
-For servers with vGPU support, the only acceptable value for
-`IMAGE_NAME` is `Ubuntu 22.04 NVGRID x86_64`. Additionally, the value
-for `FLAVOR_NAME` should be a flavor [prefixed by
-`g`](../../../reference/flavors/index.md#compute-tiers). To see all such
-flavors, you can type something like the following:
-
-```console
-$ openstack flavor list -c Name -f value | grep "^g\."
-g.8c48gb50
-g.24c120gb50
-g.48c240gb
-g.12c48gb50
-g.16c64gb50
-g.12c64gb50
-g.16c84gb50
-g.8c32gb50
-```
-
-Taking all of the above into account, an `openstack` command for
-creating a new vGPU server could look like the following:
-
-```console
-openstack server create \
-    --flavor "g.8c32gb50" \
-    --image "Ubuntu 22.04 NVGRID x86_64" \
-    --boot-from-volume 32 \
-    --network cc-net \
-    --security-group default \
-    --key-name mykey \
-    --wait \
-    my-vgpu-server
-```
-
-Before you check whether the server does have virtual GPU support, we
-recommend assigning a floating IP to it.
+=== "{{gui}}"
+    Following the instructions in the [server creation How-To
+    guide](new-server.md), go ahead and instantiate a new cloud server.
+
+    While you're at it, choose a region where you can have vGPU servers. One
+    such region would be `Stockholm/Sweden`.
+
+    ![Create new object](assets/new-vgpu-server/shot-01.png)
+
+    Also, when selecting a server profile, make sure you indicate the one
+    named _GPU_.
+
+    ![Create new object](assets/new-vgpu-server/shot-02.png)
+
+    Regarding the image your new vGPU server will be based on, your only
+    option is `Ubuntu 22.04 NVGRID x86_64`.
+
+    ![Create new object](assets/new-vgpu-server/shot-03.png)
+
+    Notice that the disaster recovery mechanism is _not_ available for this
+    type of server.
```

Comment:
I think it would be a good idea to drive home what this means, like so:

```suggestion
    Notice that the disaster recovery mechanism is _not_ available for this
    type of server.
    You will not be able to [restore the server to a snapshot](restore-srv-to-snap.md).
```

Reply:
It is a good idea, yes.